CRM/Contribute/BAO/Contribution.php

```diff
@@ -2476,6 +2480,7 @@ public function loadRelatedObjects(&$input, &$ids, $loadAll = FALSE) {
       }
     }
 
+    // These are probably no longer accessed from anywhere
```

Comment:
Add @fixme/@todo tag

Reply:
@mattwire you are right - I added this & a few more. Just thinking about it - in the past sometimes I've flagged a bit of a roadmap and even year's later people have followed it so I think I should be more aggressive with my 'todos'